.github/workflows/test.yml

```diff
@@ -100,3 +100,35 @@ jobs:
         run: |
           npm run extract-messages
           git diff --exit-code ./locale
+
+  chromatic-storybook:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Consider splitting the CI into multiple files, that job doesn't appear to depend on the rest of the jobs

Reply:
Sure - fixed 😀